### PR TITLE
updating operator framework api dependency

### PR DESCRIPTION
**Pull request by acornett21**
- Updating `operator-framework/api` and migrating to new validations, since the original is depricated

**Note:** OperatorHubV2Validator, StandardCapabilitiesValidator and StandardCategoriesValidator provide the same functionality as the deprecated OperatorHubValidator validator

### CODE REVIEW

File: internal/bundle/bundle.go [hunk 1]
```diff
@@ -44,7 +44,11 @@ func Validate(ctx context.Context, imagePath string) (*Report, error) {
 	}
 	validators := validation.DefaultBundleValidators.WithValidators(
 		validation.AlphaDeprecatedAPIsValidator,
-		validation.OperatorHubValidator,
+		// OperatorHubV2Validator, StandardCapabilitiesValidator and StandardCategoriesValidator provide the same functionality
+		// as the deprecated OperatorHubValidator validator
```
Comment (komish) [inline internal/bundle/bundle.go:48]: Nit: We probably don't need these comments here. It was useful for my review, but I don't think it needs to live in the source.

**Review by komish — APPROVED** [review]
Left a nit. Non-blocking.

/lgtm

File: internal/bundle/bundle.go (hunk 1 above)
Comment (bcrochet) [reply internal/bundle/bundle.go:48]: Agreed. Should just live in the commit message.

**Review by bcrochet — COMMENTED** [review]
Agreement on the comment.

**Review by komish — APPROVED** [review]
/lgtm

**Pull request merged**